binding/form_mapping.go

```diff
@@ -124,7 +118,12 @@ func setWithProperType(valueKind reflect.Kind, val string, structField reflect.V
 			structField.Set(reflect.New(structField.Type().Elem()))
 		}
 		structFieldElem := structField.Elem()
-		return setWithProperType(structFieldElem.Kind(), val, structFieldElem)
+		return setWithProperType(structFieldElem.Kind(), val, structFieldElem, typeField)
+	case reflect.Struct:
+		if _, isTime := structField.Interface().(time.Time); isTime {
+			return setTimeField(val, typeField, structField)
+		}
+		fallthrough
```

Comment:
why need `fallthrough`?

Reply:
To keep returning `"Unknown type"` error if any other type of `struct` is given (as it was before).

Reply:
If happen error but it is `reflect.Struct` not `Unknown type`, so I think better return other error message not `Unknown type`. thanks!